### ISSUE TITLE
The background of editorGroupHeader is red, which bugging me, and different from the screen shot

### ISSUE THREAD

**Issue by WindSoilder** [mao-santaella-rs/NightWolfTheme#29]
**Please make sure you're using the latest version of the plugin before submitting an issue**
Yeah, I'm sure on this.

**Screenshots**
![图片](https://user-images.githubusercontent.com/22256154/122715051-e729b980-d29a-11eb-978f-9a187f383d6e.png)

It seems so strange, I don't know if I'm the only one who can reproduce the issue..

So I have to twisted the theme my self to add the following configuration:
```json
"workbench.colorCustomizations": {
        "[Night Wolf [dark gray]]": {
            "editorGroupHeader.noTabsBackground": "#272727",
            "editorGroupHeader.tabsBackground": "#272727",
        }
}
```

**Comment by mao-santaella-rs** [comment]
hi again @WindSoilder thanks again, I'll be looking into this shortly too

**Comment by mao-santaella-rs** [comment]
Hi @WindSoilder this is also fixed in v1.9.0, let me know what you think

**Comment by WindSoilder** [comment]
Yeah, thanks, the bug is fixed
